internal/hub/controllers/slice_controller_ut_test.go

```diff
@@ -0,0 +1,302 @@
+/*
+ *  Copyright (c) 2022 Avesha, Inc. All rights reserved. # # SPDX-License-Identifier: Apache-2.0
```

Comment:
Copyright formatting is off, 
```
# SPDX....
```
should be on the new line. This applies to mock.go too